src/test/encoding/types.h

```diff
@@ -41,7 +41,7 @@ TYPE_FEATUREFUL(entity_inst_t)
 #include "osd/OSDMap.h"
 TYPE(osd_info_t)
 TYPE(osd_xinfo_t)
-TYPE_FEATUREFUL_STRAYDATA(OSDMap)
+TYPE_FEATUREFUL_NOCOPY(OSDMap)
```

Comment:
Sure, that was my original suggestion as I thought that was your intent originally but Sam talked me out of it :) Another change coming up...